lib/cloud/aws/policy_statements.go

```diff
@@ -223,7 +224,7 @@ func StatementForListRDSDatabases() *Statement {
 
 // StatementForS3BucketPublicRead returns the statement that
 // allows public/anonynous access to s3 bucket/prefix objects.
-func StatementForS3BucketPublicRead(s3bucketName, objectPrefix string) *Statement {
+func StatementForS3BucketPublicRead(partition, s3bucketName, objectPrefix string) *Statement {
```

Comment:
I don't think we are using this method anywhere.
We should remove it instead, but I'll do it in another PR, no need to add more context to this one.